requirements.txt

```diff
@@ -34,3 +34,8 @@ django-cors-headers
 
 # Python3 compatibility
 six
+
+#fitbit
+django-fitbit==0.3.0
```

Comment:
Remove `django-fitbit` since it is not used.

Reply:
Done